1575 logs replacing - second batch (#1065)
codenvy/che-docs: src/main/pages/che-7/administration-guide/proc_viewing-che-theia-ide-logs-on-the-cli.adoc

[id="viewing-che-theia-ide-logs-on-the-cli_{context}"]
= Viewing Che-Theia editor logs using the OpenShift CLI

> Observing Che-Theia editor logs helps get a better understanding and insight over the plug-ins loaded by the editor. This section describes how to access the Che-Theia editor logs using the OpenShift CLI (command-line interface).

Reviewer: ```suggestion
Observing Che-Theia editor logs helps to get a better understanding and insight of the plug-ins loaded by the editor. This section describes how to access the Che-Theia editor logs using the OpenShift CLI (command-line interface).
```

.Prerequisites

ifeval::["{project-context}" == "che"]

> * {prod-short} is deployed in an OpenShift cluster. This can be validated by observing OpenShift logs. See link:https://www.eclipse.org/che/docs/che-7/installing-che-on-openshift-4-from-operatorhub/#viewing-the-state-of-the-che-cluster-deployment-using-openshift-4-cli-tools_installing-che-on-openshift-4-from-operatorhub[Viewing the state of the {prod-short} cluster deployment using OpenShift 4 CLI tools].

Reviewer: ```suggestion
* {prod-short} is deployed in an OpenShift cluster. Verify the state of the deployment in the OpenShift logs. See link:https://www.eclipse.org/che/docs/che-7/installing-che-on-openshift-4-from-operatorhub/#viewing-the-state-of-the-che-cluster-deployment-using-openshift-4-cli-tools_installing-che-on-openshift-4-from-operatorhub[Viewing the state of the {prod-short} cluster deployment using OpenShift 4 CLI tools].
```

endif::[]
ifeval::["{project-context}" == "crw"]

> * {prod-short} is deployed in an OpenShift cluster. This can be validated by observing OpenShift logs. See link:{prod-ig-url}installing-{prod-id-short}-on-ocp-4_crw#viewing-the-state-of-the-{prod-id-short}-cluster-deployment-using-openshift-4-cli-tools_installing-{prod-id-short}-on-openshift-4-from-operatorhub[Viewing the state of the {prod-short} cluster deployment using OpenShift 4 CLI tools].

Reviewer: ```suggestion
* {prod-short} is deployed in an OpenShift cluster. Verify the state of the deployment in the OpenShift logs. See link:{prod-ig-url}installing-{prod-id-short}-on-ocp-4_crw#viewing-the-state-of-the-{prod-id-short}-cluster-deployment-using-openshift-4-cli-tools_installing-{prod-id-short}-on-openshift-4-from-operatorhub[Viewing the state of the {prod-short} cluster deployment using OpenShift 4 CLI tools].
```

endif::[]
* A workspace is created.
* User is located in a {prod-short} installation namespace.

.Procedure

. Obtain the list of the available pods:
+
----
$ oc get pods
----
+
.Example
[subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
----
$ oc get pods
NAME                                              READY  STATUS   RESTARTS  AGE
{prod-deployment}-9-xz6g8                                 1/1    Running  1         15h
workspace0zqb2ew3py4srthh.go-cli-549cdcf69-9n4w2  4/4    Running  0         1h
----

> . Obtain the list of the available containers in the particular pod:

Reviewer: ```suggestion
. Obtain the list of the available containers in the particular Pod:
```
